cli/src/templates/base/package.json.ejs

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "android": "expo start --android",
 	<% if (props.packageManager === "bun") { %>
-		"format": "bun run eslint '**/*.{js,jsx,ts,tsx}' --fix && bun run prettier '**/*.{js,jsx,ts,tsx,json}' --write",
+		"format": "bun run eslint **/*.{js,jsx,ts,tsx} --fix && bun run prettier **/*.{js,jsx,ts,tsx,json} --write",
```

Comment:
We do formatting like this elsewhere so I wonder if this approach would resolve the issue. @catalinmiron any thoughts on this?

```suggestion
		"format": "bun run eslint \"**/*.{js,jsx,ts,tsx}\" --fix && prettier \"**/*.{js,jsx,ts,tsx,json}\" --write",
```

Reply:
done

Reply:
Not really, when copying the templates, we're running `npx` yarn dlx` `bunx` `pnpmx` via [getPackageManager.ts](https://github.com/danstepanov/create-expo-stack/blob/beta/cli/src/utilities/getPackageManager.ts). and run `eslint` globally. This change should affect the code after `noInstall=false`.  I think that this is a valid change.

PS: Haven't tested locally but @kratos-respawned could you please share the output of both `noInstall` true and false?

Thanks!